src/core/styles/_constants.scss

```diff
@@ -40,21 +40,15 @@ $color-purple-100: hsl(263deg 55% 83%); // #cebceb
 $color-purple-50: hsl(260deg 47% 96%); // #f4f1fa
 
 // Blue colors
-$color-mentorship-header: hsl(198deg 86% 55%); // #2bb4ef
-$color-mentorship-blue: hsl(198deg 100% 80%); // #9be1ff
-$color-mentorship-blue-light: hsl(198deg 95% 62%); // #46c4fa
-
+$color-blue-100: hsl(198deg 100% 80%); // #9BE1FF
+$color-blue-200: hsl(198deg 95% 62%); // #41C3FA
```

Comment:
MD says that it's `300` color - 
![image](https://github.com/user-attachments/assets/17cf5dc9-503f-40a9-b584-41a823fae7ff)

And lets write comments in lower case

Reply:
Done